portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.data;
+
+public class Comment {
+    private final String name, comment;
+    private final long id, timestamp;
```

Comment:
nothing wrong with it, but it is generally more readable to have one declaration per line https://google.github.io/styleguide/javaguide.html#s4.8.2-variable-declarations